### DISCUSSION TITLE
Is there a better way to get the index from the repeater?

### DISCUSSION THREAD

**Discussion by joaovdiasb** [filamentphp/filament#4002] [Help] [answered]
![CleanShot 2022-09-13 at 13 52 04](https://user-images.githubusercontent.com/51106311/189960798-d7b766a9-e91b-445a-b244-5b11774a7876.png)
![CleanShot 2022-09-13 at 13 53 11@2x](https://user-images.githubusercontent.com/51106311/189961052-3f481acf-43f7-46ab-9179-0660ea0cd0a9.png)

I'm doing it this way, but it's very verbose

**Comment by danharrin** [comment, marked as answer]
Yeah that approach is correct

You can clean it up a bit:

`array_search($itemKey, array_keys($get('../../pavimentos')))`

I'm not sure how we can make this easier really

**Reply by joaovdiasb** [reply]
Thank you, better this way

**Comment by sneycampos** [comment]
Hey, i'm late but i was looking for the same solution and found this:

```PHP
->itemLabel(function () {
    static $position = 1;
    return $position++;
})
```
